examples/gno.land/p/samourai/daokit/messages.gno

```diff
@@ -0,0 +1,112 @@
+package daokit
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/moul/md"
+)
+
+type ExecutableMessage interface {
+	String() string
+	Type() string
+}
+
+type MessageHandler interface {
```

Comment:
```suggestion
type Handler interface {
```

Reply:
i kept ActionHandler that is more explicit IMO